freqtrade/rpc/rpc.py

```diff
@@ -135,9 +135,11 @@ def _rpc_trade_status(self) -> List[Dict[str, Any]]:
                 trade_dict = trade.to_json()
                 trade_dict.update(dict(
                     base_currency=self._freqtrade.config['stake_currency'],
-                    close_profit=fmt_close_profit,
+                    close_profit=trade.close_profit or None,
+                    close_profit_perc=fmt_close_profit,
```

Comment:
@hroff-1902 
Should this maybe use `close_profit_pct`? (oll other occurances of _perc too, obviously).

that would put it in line with #3377  - and honestly, i find "pct" a better shortcut for percentage than "perc"...

Reply:
> @hroff-1902
> Should this maybe use `close_profit_pct`? (oll other occurances of _perc too, obviously).
> 
> that would put it in line with #3377 - and honestly, i find "pct" a better shortcut for percentage than "perc"...

agree, that's the better suffix for these names...